packages/core/src/shared/lsp/manifestResolver.ts

```diff
@@ -69,7 +69,6 @@ export class ManifestResolver {
 
             const localManifest = await this.getLocalManifest(true).catch(() => undefined)
             if (localManifest) {
-                localManifest.location = 'remote'
                 return localManifest
```

Comment:
Why unset location=remote ? The reason I set that field, is because this logic is in the `fetchRemoteManifest` function. `this.getLocalManifest()` is being used for code re-use.